research/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: ami-ml
```

Comment:
I was thinking more of a top level environment, located at the root of the project, to be used by the whole project, by the other members of the project also